src/families/ethereum/modules/compound.js

```diff
@@ -396,11 +396,16 @@ export function prepareTokenAccounts(
   return subAccounts.concat(implicitCTokenAccounts);
 }
 
-const cdaiToDaiOpMapping: { [_: OperationType]: ?OperationType } = {
+const ctokenToGeneratedTokenOpMapping: { [_: OperationType]: ?OperationType } = {
```

Comment:
@stephenjust can you run `yarn prettier` ? thanks

Reply:
Looks like Junichi got it. Thanks!